staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -321,7 +322,8 @@ func CreateListener(network, addr string) (net.Listener, int, error) {
 	if len(network) == 0 {
 		network = "tcp"
 	}
-	ln, err := net.Listen(network, addr)
+	config := &net.ListenConfig{Control: reusePort}
```

Comment:
I think `permitPortReuse` would be an even more obvious name.

Reply:
Renamed.